### ISSUE TITLE
a11y: High contrast dropdown selection visibility/usability issue

### ISSUE THREAD

**Issue by jgage176** [microsoft/fluentui#20462]
### Environment Information

- **Package version(s)**: Win 10
- **Browser and OS versions**: Edge Version 94.0.992.47

### Describe the issue:

When only two options are provided for a dropdown and one is selected, when doing high contrast (both black or white) it is confusing and hard to tell what is happening when using the keyboard to navigate. Here is a picture of the first item highlighted:

![image](https://user-images.githubusercontent.com/29739628/139925405-52fd432f-4957-48c6-8c06-8d9bb33cc391.png)

Here is when the second item is highlighted:

![image](https://user-images.githubusercontent.com/29739628/139925557-c52cc4d6-8a82-44ac-bf79-5a7dd98541d5.png)

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/jgage176/pen/JjyOzKK

#### Actual behavior:

When pressing down arrow key, keyboard focus is not visible.

Similar issue is also observed in both Black and white high contrast.

High contrast  user will face difficulty in finding the keyboard focus on the screen.

#### Expected behavior:

When pressing down arrow key, keyboard focus should be visible.

### Documentation describing expected behavior

All features should be useable by keyboard, even when in high contrast mode.

**Comment by khmakoto** [comment]
Hi @jgage176, I can confirm that this is happening, thanks for filing this!

**Comment by paivybhav99** [comment]
Hey guys, any updates on this issue? facing the same issue.
